commands/bot.rb

```diff
@@ -17,7 +17,11 @@
   description 'Displays the version information.'
   usage 'version'
   on do
-    reply "Scarlet v#{Scarlet::Version::STRING}"
+    str = "Scarlet v#{Scarlet::Version::STRING}"
+    if commit = ENV['SCARLET_COMMIT']
+      str += " commit #{fmt.commit_sha(commit)}"
```

Comment:
http://stackoverflow.com/questions/4684446/why-is-the-shovel-operator-preferred-over-plus-equals-when-building-a